samwise.js

```diff
@@ -6,50 +6,46 @@
  */
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?